README.md

```diff
@@ -5,29 +5,31 @@
 ## About this Repository
 This repository contains all issues that need to be addressed regarding the hotlines of the Corona-Warn-App. The App hotline is for technical questions regarding the Corona-Warn-App. The TAN hotline serves requests to issue TANs for the Corona-Warn-App. The national telephone number is to be used when calling from within Germany. The international number is available for calls originating outside Germany. The operating hours of the app and TAN hotlines are listed in the [Support](https://www.coronawarn.app/en/faq/#support) section of the [Corona-Warn-App website](https://www.coronawarn.app/).
 
-Hotline | App | TAN
---- | --- | ---
-National | [0800 7540001](tel:08007540001) | [0800 7540002](tel:08007540002)
-International | [+49 30 498 75401](tel:+493049875401) | [+49 30 498 75402](tel:+493049875402)
+*Following the release of CWA 3.0 on Jan 18, 2023 the TAN Hotline is no longer necessary and shuts down on Jan 31, 2023. See the blog announcement [Version 3.0: CWA warnings now possible after positive self-test](https://www.coronawarn.app/en/blog/2023-01-18-cwa-3-0/) for further details.*
```

Comment:
May I suggest phrasing it in a way which only requires a change once?

```suggestion
*Following the release of CWA 3.0 on Jan 18, 2023 the TAN Hotline is no longer necessary and is no longer available from Jan 31, 2023 on. See the blog announcement [Version 3.0: CWA warnings now possible after positive self-test](https://www.coronawarn.app/en/blog/2023-01-18-cwa-3-0/) for further details.*
```

Reply:
@Ein-Tim 

Both the original text and your suggestion are grammatically equivalent, so I don't plan to change. Thank you for taking the time to check.

Reply:
@MikeMcC399 

Would change your text again after 31. Jan? If yes, I suggest using any other text which does not require another change.